ownership-chain/pallets/living-assets-ownership/src/types.rs

```diff
@@ -0,0 +1,16 @@
+//! Types and type aliases used in LivingAssetsOwnership pallet.
```

Comment:
1. aliased types so that they are defined in one place
2. explicitly specified type for `AssetId`, similar to `CollectionId`